test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1025,29 +1025,6 @@ BOOST_AUTO_TEST_CASE(blockchain)
 	ABI_CHECK(callContractFunctionWithValue("someInfo()", 28), encodeArgs(28, u256("0x1212121212121212121212121212121212121212"), 7));
 }
 
-BOOST_AUTO_TEST_CASE(now)
```

Comment:
Do we have a test like this somewhere now?

Reply:
@axic Good point. I'll add one back that doesn't use now.